etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/metadata/PartnerMetadataStorage.java

```diff
@@ -0,0 +1,14 @@
+package gov.hhs.cdc.trustedintermediary.etor.metadata;
+
+/** Interface to store and retrieve our partner-facing metadata. */
+public interface PartnerMetadataStorage {
+    PartnerMetadata readMetadata(String uniqueId);
+
+    /**
+     * This method will do "upserts". If the record doesn't exist, it is created. If the record
```

Comment:
Thinking about record add vs update: is there a risk of overwriting metadata that we would need to maintain as part of the official record of submitted orders?